internal/server/repository_worker_test.go

```diff
@@ -691,12 +695,24 @@ func TestListWorkers_WithExcludeShutdown(t *testing.T) {
 		server.NewWorker(scope.Global.String(),
 			server.WithName(worker3.GetName()),
 			server.WithAddress(worker3.GetAddress()),
+			server.WithReleaseVersion("Boundary v.0.11"),
 			server.WithOperationalState(server.ShutdownOperationalState.String())),
 		server.WithPublicId(worker3.GetPublicId()))
 	require.NoError(err)
 	result, err = serversRepo.ListWorkers(ctx, []string{scope.Global.String()}, server.WithExcludeShutdownWorkers(true))
 	require.NoError(err)
 	require.Len(result, 0)
+
+	// Upsert without a release version or state and expect to get a hit- test backwards compatibility
```

Comment:
Can we add tests to cover each of the `or` parts of the where clause?

Reply:
The existing test should cover the nil version; I've added one for an empty string version. Thanks!